armsrc/appmain.c

```diff
@@ -1271,21 +1276,36 @@ void UsbPacketReceived(uint8_t *packet, int len) {
 			size_t len = 0;
 			uint32_t startidx = c->arg[0];
 			uint32_t numofbytes = c->arg[1];
+			uint8_t fast = c->arg[2];
+
 			// arg0 = startindex
 			// arg1 = length bytes to transfer
 			// arg2 = RFU
+
+
+			if (fast) {
+				FlashInit(1);
+				//FlashInit();
```

Comment:
Only **fast** that has a FlashInit ?

Reply:
Yes with the ReadData logic. No I'd it is rewritten with ReadDataCont which would avoid flashinit/flashtop for each reading cycle.